pkg/source/source_test.go

```diff
@@ -350,7 +350,7 @@ var _ = Describe("Source", func() {
 						if eventCount == 0 {
 							<-unblock
 						}
-						eventCount += 1
+						eventCount++
```

Comment:
I don't think that this change should be apart of this PR.

Reply:
The lint check was failing because of `eventCount += 1`. I figured it would break the CI if I did not make the minor change. Thoughts on how to handle it?